client/ayon_silhouette/api/plugin.py

```diff
@@ -0,0 +1,191 @@
+from ayon_core.pipeline import (
+    Creator,
+    LoaderPlugin,
+    CreatedInstance,
+    AYON_INSTANCE_ID,
+    AVALON_INSTANCE_ID,
+    CreatorError,
+)
+from ayon_core.lib import BoolDef
+from . import lib
+
+import fx
```

Comment:
This should probaly be above all other imports here.

Reply:
https://github.com/ynput/ayon-silhouette/pull/1/commits/bebddaee422a8436c500d7dfdd93a73f79c54d84